Assets/Scripts/UI/MenuManager.cs

```diff
@@ -0,0 +1,49 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+public class MenuManager : MonoBehaviour
+{
+    public GameObject PauseMenuByEsc;
+
+    public static bool IsPaused;
+    // Start is called before the first frame update
+    void Start()
```

Comment:
Empty method body can be condensed to one line: `Start() { }` or the method can be removed